noq_django/rest_api/api/test/test_host_handle_booking_api.py

```diff
@@ -156,8 +161,37 @@ def test_batch_accept_bookings(self):
         parsed_response = json.loads(response.content)
         self.assertEqual(len(parsed_response), 0)
 
+    def test_pending_bookings_exclude_past_start_date(self):
+
+        # Create one booking with a past start date
+        Booking.objects.bulk_create([
+            Booking(
+                start_date=datetime.now().date() - timedelta(days=1),  # Past start date
+                end_date=datetime.now().date() + timedelta(days=1),
+                product=Product.objects.get(name="room"),
+                user=Client.objects.first(),
+                status=BookingStatus.objects.get(id=State.PENDING)
+            ),
+            Booking(
+                start_date=datetime.now().date() + timedelta(days=1),  # Future start date
+                end_date=datetime.now().date() + timedelta(days=2),
+                product=Product.objects.get(name="room"),
+                user=Client.objects.last(),
+                status=BookingStatus.objects.get(id=State.PENDING)
+            )
+        ])
+
```

Comment:
You could add a check here to verify that you have 6 pending bookings in the DB.